api/database/seeders/PoolTestSeeder.php

```diff
@@ -43,25 +43,196 @@ public function run()
                 'closing_date' => config('constants.far_future_date'),
                 'publishing_group' => PublishingGroup::IAP->name,
             ],
-        ];
+            
+            //  IT-01  - Draft Job
+            [
```

Comment:
The doc says this should be all filled out but there are some missing fields.
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/ec5452dc-9db2-49f6-99f4-565a208e327d)
Same for the IT-02 pool.